src/ophyd_async/core/_signal.py

```diff
@@ -110,6 +110,9 @@ async def connect(
         ), "this assert is for type analysis and will never fail"
         await self._connect_task
 
+    def get_backend(self) -> SignalBackend:
+        return self._backend
+
```

Comment:
I explicitly wanted to hide the backend from the public interface, as the only place that uses this is in the mock signal backend access above, so it shouldn't be public for everything else

Reply:
Okay cool.